config/jobs/kubernetes/cloud-provider-azure/cloud-provider-azure-config.yaml

```diff
@@ -158,3 +206,107 @@ periodics:
       - --timeout=420m
       securityContext:
         privileged: true
+
+- interval: 8h
+  # ci-cloud-provider-azure-autoscaling runs node autoscaling tests periodically.
+  name: ci-cloud-provider-azure-autoscaling
+  labels:
+    preset-service-account: "true"
+    preset-cloudprovider-azure-cred: "true"
+    preset-dind-enabled: "true"
+  decorate: true
+  extra_refs:
+  - org: kubernetes
+    repo: cloud-provider-azure
+    base_ref: master
+  path_alias: k8s.io/cloud-provider-azure
+  skip_submodules: true
+  spec:
+    containers:
+    - image: gcr.io/k8s-testimages/kubekins-e2e:v20190315-49d62eb51-master
+      args:
+      - --job=$(JOB_NAME)
+      - --repo=k8s.io/kubernetes=v1.14.0-alpha.1
```

Comment:
Does this need to be updated to match support for v1.14.0-rc.1?

Reply:
better to match, but a little patch version difference is unavailable since those are in different repos. Fortunately, it's still working with patch version differences.